### DISCUSSION TITLE
Does the installation supports python 3.11.4?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#5744] [Q&A] [answered]
Does the installation support python 3.11.4? Also, are all the functions compatible with python 3.11.4?

**Comment by squidfunk** [comment, marked as answer]
Yes, Python 3.11.x should be supported. If you encounter errors, please report them by following our [bug reporting guide](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).

**Comment by mendax1234** [comment]
Sure, thanks for your reply!
